sanic/__init__.py

```diff
@@ -1,7 +1,6 @@
+from sanic.__version__ import __version__
 from sanic.app import Sanic
 from sanic.blueprints import Blueprint
 
 
-__version__ = "19.6.2"
```

Comment:
Why not just keep it here ?

Reply:
Moving it to a dedicated `__version__` file can be handy for a lot of tooling down the line if required. It can be useful if we want to automate the process of generating the version update as well.

Reply:
Working on adding a `make release` support that will take care of everything except for creating a PR and tagging it. (Creating a PR is a possible addition I will look into)